api/_globals/performance.json

```diff
@@ -67,6 +74,16 @@
             "ie": {
               "version_added": false
             },
+            "nodejs": [
+              {
+                "version_added": "16.0.0"
+              },
+              {
+                "version_added": "11.7.0",
```

Comment:
How did you determine this version?

Reply:
see https://github.com/mdn/browser-compat-data/pull/25014, the first version that support worker without behind a flag is v11.7.0